es-design-system/pages/organisms/es-form.vue

```diff
@@ -135,50 +166,70 @@ export default {
                     arguments: 'function',
                     description: 'a function with access to string; expected to return a boolean',
                 },
                 {
-                    name: 'vuelidateMinValue',
+                    name: vuelidateKeys.MIN_VALUE,
                     arguments: 'minimum',
                     description: 'check if number is greater than minimum',
                 },
                 {
-                    name: 'vuelidateMaxValue',
+                    name: vuelidateKeys.MAX_VALUE,
                     arguments: 'maximum',
                     description: 'check if number is less than maximum',
                 },
                 {
-                    name: 'vuelidateMinLength',
-                    arguments: 'minimum',
-                    description: 'check if string has at least the mimimum characters count',
+                    name: vuelidateKeys.MIN_LENGTH,
+                    arguments: '',
+                    description: 'This field must be at least 8 characters',
                 },
                 {
-                    name: 'vuelidateMaxValue',
+                    name: vuelidateKeys.MAX_LENGTH,
                     arguments: 'maximum',
                     description: 'check if string has less than the maximum characters count',
                 },
                 {
-                    name: 'vuelidateNumeric',
+                    name: vuelidateKeys.NUMERIC,
                     arguments: '',
                     description: 'check if the value is a number',
                 },
                 {
-                    name: 'vuelidateEmail',
+                    name: vuelidateKeys.EMAIL,
                     arguments: '',
                     description: 'check if the string is a valid email address',
                 },
                 {
-                    name: 'vuelidatePhone',
+                    name: vuelidateKeys.PHONE,
                     arguments: '',
                     description: 'check if the string is a valid US phone number',
                 },
+                {
+                    name: vuelidateKeys.HAS_NUMBER,
+                    arguments: '',
+                    description: 'check if the string contains a number',
+                },
+                {
+                    name: vuelidateKeys.HAS_SPECIAL_CHARACTER,
+                    arguments: '',
+                    description: 'check if the string contains a special character',
+                },
+                {
+                    name: vuelidateKeys.HAS_UPPERCASE_LETTER,
+                    arguments: '',
+                    description: 'check if the string contains an uppercase character',
+                },
+                {
+                    name: vuelidateKeys.HAS_LOWERCASE_LETTER,
+                    arguments: '',
+                    description: 'check if the string contains a lowercase character',
+                },
```

Comment:
I like that these are individual so we can use them that way but I wonder for this use case we should just have a single `vuelidateKeys.COGNITO_PASSWORD` or something which contains all validators needed to validate a cognito password and their definitions.